modules/gapi/src/streaming/onevpl/source_priv.cpp

```diff
@@ -47,7 +51,7 @@ enum {
 };
 
 GSource::Priv::Priv() :
-    mfx_handle(MFXLoad()),
+//    mfx_handle(MFXLoad()),
```

Comment:
???

Reply:
`mfx_handle` is  a global variable by now. It was implemented to overcome oneVPL  issue with cloning session:

all VPL operations (decode, VPP) must belong to `mfxSession` context. this mfxSession holds DevicePtr, bitstream settings and some other infrastructural data. So to queue VPP preprocessing operation we must create different `mfxSession` wich will share some settings with decoding ` mfxSession` and has different another's.  At least both session must share the same VPL Library Implementation and version which selected in VPLSource configuration step.
The ideas was:
pass decode session object as MediaFrameAdapter data, then PreprocEngine extract it from MediaFrameAdapter in `is_applicable` method and perform `mfxSessionClone` in order to share the MAJOR session params: Implementation, Version, Decoder ID and so; then operates with VPP cloned session and forget about decode session (to avoid huge operation for mfxLoad and choosing implementation based on selected criteria, which may be diffrent from producing VPL source  and so on).

But i faced with some issue in oneVPL implementation which rejects cloning session due to some inner bug. 
To overcome this issue i made `mfx_handle` as global variable (temporary) with limitation "only single VPL Source can exist" then i just use this global `mfx_handle` state to create a NEW session (not as clone) from it's state.
After new VPL version releases we will reactivate idea with Cloning Session and revert such commented lines back